src/ProjectNeon/Assets/Features/Battle/Cards/SelectCardTargets.cs

```diff
@@ -46,7 +46,7 @@ private void BeginSelection()
         cardPresenter.Set(_selectedCard, () => { });
         uiView.SetActive(true);
 
-        var possibleTargets = battleState.GetPossiblePlayerTargets(_selectedCard.TargetGroup, _selectedCard.TargetScope);
+        var possibleTargets = battleState.GetPossiblePlayerTargets(_selectedCard.Actions[0].Group, _selectedCard.Actions[0].Scope);
```

Comment:
Add a puzzle here for selected targets for actions after the first.

Reply:
@SilasReinagel Done!